umap/static/umap/js/umap.js

```diff
@@ -65,9 +65,16 @@ L.U.Map.include({
   ],
 
   initialize: function (el, geojson) {
-    if (geojson.properties && geojson.properties.locale)
```

Comment:
Not sure how critical it is but we are not testing anymore if `geojson` has `properties` first.

[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) might be a way to handle the case without overcharging the syntax but it's for relatively recent browsers (2020).

Reply:
Yep, I removed it be cause all the following lines were consuming properties without this check, so at this point…